haskell/src/Monocle/Backend/Janitor.hs

```diff
@@ -40,6 +42,22 @@ updateIdentsOnChanges = do
   indexName <- getIndexName
   doUpdateIdentsOnChanges indexName (updateAuthor index)
 
+--- Dedicated reduced data type of EChange with only Author fields
```

Comment:
I guess you noticed performance issue with the full type. Didn't you also mention an option to also filter the fields returned by the elastic api?

Reply:
I think this PR helps to improve the performance of the update-idents command thanks !
https://github.com/change-metrics/monocle/pull/757